app/src/main/java/in/ac/bits_pilani/goa/ard/utils/AHC.java

```diff
@@ -19,7 +20,7 @@
     /**
      * Default Log tag for project.
      */
-    public static final String TAG = APPLICATION_ID;
+    public static final String TAG = "MAC_ARD";
```

Comment:
Should we instead use another variable `SHORT_TAG`? The previous one is standard way of logging, I guess.